README.md

```diff
@@ -374,7 +374,25 @@ const account = await starknet.getAccountFromAddress(
 );
 ```
 
-The balance of an account can be checked using:
+### Predeployed accounts can be checked using:
```

Comment:
Let's keep `##` and `###` for titles more than real sentences (so here it could be something like `### Fetch predeployed accounts`).